beacon-chain/db/filesystem/cache.go

```diff
@@ -67,48 +69,70 @@ func (s *blobStorageCache) Summary(root [32]byte) BlobStorageSummary {
 	return s.cache[root]
 }
 
-func (s *blobStorageCache) ensure(key [32]byte, slot primitives.Slot, idx uint64) error {
-	if idx >= fieldparams.MaxBlobsPerBlock {
+func (s *blobStorageCache) ensure(ident blobIdent) error {
+	if ident.index >= fieldparams.MaxBlobsPerBlock {
 		return errIndexOutOfBounds
 	}
 	s.mu.Lock()
 	defer s.mu.Unlock()
-	v := s.cache[key]
-	v.slot = slot
-	if !v.mask[idx] {
+	v := s.cache[ident.root]
+	v.epoch = ident.epoch
+	if !v.mask[ident.index] {
 		s.updateMetrics(1)
 	}
-	v.mask[idx] = true
-	s.cache[key] = v
+	v.mask[ident.index] = true
+	s.cache[ident.root] = v
 	return nil
 }
 
-func (s *blobStorageCache) slot(key [32]byte) (primitives.Slot, bool) {
+func (s *blobStorageCache) get(key [32]byte) (BlobStorageSummary, bool) {
```

Comment:
Would this not be better named `summary` since it returns the `BlobStorageSummary`?
```suggestion
func (s *blobStorageCache) summary(key [32]byte) (BlobStorageSummary, bool) {
```

Reply:
`get` is an internal method that also indicates if the value is present in the cache for initialization purposes, whereas the public `Summary` always returns a `BlobStorageSummary` that can be used by code outside the storage package.